packages/openapi3/src/cli/actions/compile/utils/docs.ts

```diff
@@ -0,0 +1,69 @@
+export function generateDocs(doc: string | string[]): string {
```

Comment:
I copied these utils directly from https://github.com/Azure/autorest/blob/main/packages/extensions/openapi-to-typespec/src/utils/docs.ts